product-service/populateData.js

```diff
@@ -0,0 +1,47 @@
+const AWS = require('aws-sdk');
+const { books } = require('./src/content/books.json');
+const { count } = require('./src/content/count.json');
+
+AWS.config.update({
+  region: 'eu-central-1'
```

Comment:
it's not so good idea to hardcode such configuration values directly in code, especially in files where you work with data populating